source/features/download-folder-button.tsx

```diff
@@ -22,11 +22,22 @@ function add(folderDropdown: HTMLElement): void {
 	);
 }
 
+function add({parentElement: deleteDirectoryItem}: HTMLAnchorElement): void {
+	const item = deleteDirectoryItem!.cloneNode(true);
+	const link = item.firstElementChild as HTMLAnchorElement;
+	const downloadUrl = new URL('https://download-directory.github.io/');
+	downloadUrl.searchParams.set('url', location.href);
+	link.href = downloadUrl.href;
+	link.textContent = 'Download folder';
```

Comment:
This also clears a redundant `span` which happens to also be what sets the text color to red on "Delete directory"


<img width="493" alt="Screenshot 11" src="https://github.com/refined-github/refined-github/assets/1402241/231c0f22-fca2-4130-9bdb-e9ce8cfdd0da">

Reply:
```suggestion
	link.textContent = 'Download directory';
```